staging/src/k8s.io/kubectl/pkg/cmd/cmd.go

```diff
@@ -473,7 +473,6 @@ func NewKubectlCommand(in io.Reader, out, err io.Writer) *cobra.Command {
 	}
 
 	flags := cmds.PersistentFlags()
-	flags.SetNormalizeFunc(cliflag.WarnWordSepNormalizeFunc) // Warn for "_" flags
```

Comment:
`cliflag.WarnWordSepNormalizeFunc` seems to be different from `cliflag.WordSepNormalizeFunc` at 479 according to their names. Could you help me understand why this line could be removed?

Reply:
The difference between `WarnWordSepNormalizeFunc` and `WordSepNormalizeFunc` is that the former will output a warning. The function of both is order to set `flags.normalizeNameFunc`. I think the latter(479) code will override the previous(476) result.It's only my personal opinion and i found when i removed it, i can still use `kubectl` normally.